### ISSUE TITLE
Support for React 18

### ISSUE THREAD

**Issue by arndom** [EitanElbaz/ReactGrandTour#6]
There is a npm error installing the package:

```
npm ERR! node_modules/react
npm ERR!   react@"^18.2.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.2" from react-grand-tour@0.14.0
```

**Comment by EitanElbaz** [comment]
Will take a look very soon thank you

**Comment by EitanElbaz** [comment]
Resolved in 0.16.0
